intellij-plugin-structure/structure-intellij/src/test/kotlin/com/jetbrains/plugin/structure/intellij/platform/ModuleBeanParsingTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.jetbrains.plugin.structure.intellij.platform
+
+import com.jetbrains.plugin.structure.intellij.beans.ModuleBean
+import com.jetbrains.plugin.structure.intellij.extractor.ModuleUnmarshaller
+import junit.framework.TestCase.assertNotNull
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertThrows
+import org.junit.Test
+import java.nio.file.Path
+import javax.xml.bind.UnmarshalException
+
+class ModuleBeanParsingTest {
```

Comment:
why do we rely on functionality that is still WIP and only available in 242+ AFAIK?

Reply:
Using just the resource paths and module names. No dependency management is taken from this file.